### PR TITLE
towards completion of Issue#23

### PR DESCRIPTION
**Pull request by ghost**
Sample commit part -1 
Towards completion of Issue #23 

Added RPCC, Swedish registry,Norther Ireland Registry, NFR

### CODE REVIEW

File: truebase/things/NFR.cdb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+title NFR
+type registry
+standsFor National Firefighter Registry
+appeared 2018
+description The National Childhood Cancer Registry (NCCR) is a rapidly growing public health surveillance data resource. Its primary goal is to provide a platform to better understand the causes, outcomes, effective treatments, and later effects of cancer among children, adolescents, and young adults in the U.S.
```
Comment (breck7) [inline truebase/things/NFR.cdb]: Good find on the NFR!

This description does not match NFR.

Comment (breck7) [reply truebase/things/NFR.cdb]: NM, TIL I can make updates to a Pull Request right in GitHub!

**Comment by breck7** [comment]
Great work @SRS-WRKS ! There were a few nits that I went ahead and updated. I did not know until today that I can make edits to PRs.

**Comment by ghost** [comment]
Hi,


3 Mar 2023, 21:45 by ***@***.***:

>
>
>
>
> Great work > @SRS-WRKS <https://github.com/SRS-WRKS>>  ! There were a few nits that I went ahead and updated. I did not know until today that I can make edits to PRs.
>
>
Pull requests and patch reviews are both named PR


It depends.


Different open source people prefer different things. I am sorry :)

**Pull request merged**